.github/workflows/update-render-staging-service.yml

```diff
@@ -0,0 +1,34 @@
+name: Update render staging service
+
+on:
+  create:
+    branches:
+      - "release/*.*.*"
+
+jobs:
+  update-render:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Extract version
+        run: |
+          echo "RELEASE_VERSION=${GITHUB_REF_NAME#release/}" >> $GITHUB_ENV
+      - name: Update render web service
+        run: |
+          export RENDER_API_STATUS_CODE=$(curl --request PATCH \
```

Comment:
nit: remove export (don't need to export), -o (we can look at the output in the log console).

Reply:
Do we want the action fail if the status code is not 200?

Reply:
Yeah, let's fail the action in this case.

Reply:
If we don't use export, how can we get the single status_code without the response and use it in if-block followed?

Reply:
# sh 1.sh
a=123
echo $a

% sh 1.sh 
123


https://www.baeldung.com/linux/bash-variables-export
"export" is used for setting up environment variables.

Reply:
Done.